packages/babel-cli/bin/babel/file.js

```diff
@@ -100,7 +100,7 @@ module.exports = function (commander, filenames, opts) {
     results = [];
 
     _.each(filenames, function (filename) {
-      if (!fs.existsSync(filename)) return;
+      if (!pathExists.sync(filename)) return;
```

Comment:
Should I open a new PR with the fix? Not sure how GitHub will work with new changes on the branch.